.eslintignore

```diff
@@ -1,2 +1,5 @@
+!.eslintrc.js
 addon/StudyUtils.jsm
 addon/webextension/vendor/*
+dist
+test
```

Comment:
/test/ is ignored for now because there are a bunch of errors I didn't have time to dig through.